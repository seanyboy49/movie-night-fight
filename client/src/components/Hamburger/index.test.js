import { shallow } from 'enzyme'
import Burger from '.'

describe('<Burger />', () => {
  it('should render', () => {
    const wrapper = shallow(<Burger />)

    expect(wrapper).toMatchSnapshot()
  })
})
