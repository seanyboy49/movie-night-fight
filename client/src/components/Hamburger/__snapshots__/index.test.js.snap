// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Burger /> should render 1`] = `
<styled.button
  isOpen={false}
  onClick={[Function]}
>
  <div />
  <div />
  <div />
</styled.button>
`;
